apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -70,6 +70,7 @@ class ReviewTab extends Component {
       serverLevelId,
       serverScriptId
     } = getStore().getState().pageConstants;
```

Comment:
may be out of scope, but would you mind moving these to be passed in as props via redux?

Reply:
Curious about this in general -- since these are constants (ie, shouldn't change), do we still prefer they be passed in as props?

Reply:
From what I've seen, getStore.getState generally only gets called for libraries that can't be connected. React components generally get values like this by being connected and passing via props. I don't know the exact reason why we don't tend to call getStore.getState from within a component, but I'd generally be surprised to find something controlling a component by anything besides props.

Reply:
I have one more follow-up in the works after the functional change. I'll plan to do this as part of that change unless someone else gets to it before me.